examples/Remarketing/UploadOfflineConversions.php

```diff
@@ -0,0 +1,157 @@
+    const CONVERSION_VALUE = 'INSERT_CONVERSION_VALUE_HERE';
+
+    public static function main()
+    {
+        // Either pass the required parameters for this example on the command line, or insert them
+        // into the constants above.
+        $options = (new ArgumentParser())->parseCommandArguments([
+            ArgumentNames::CUSTOMER_ID => GetOpt::REQUIRED_ARGUMENT,
+            ArgumentNames::CONVERSION_ACTION_ID => GetOpt::REQUIRED_ARGUMENT,
+            ArgumentNames::GCL_ID => GetOpt::REQUIRED_ARGUMENT,
+            ArgumentNames::CONVERSION_TIME => GetOpt::REQUIRED_ARGUMENT,
+            ArgumentNames::CONVERSION_VALUE => GetOpt::REQUIRED_ARGUMENT
+        ]);
+
+        // Generate a refreshable OAuth2 credential for authentication.
+        $oAuth2Credential = (new OAuth2TokenBuilder())->fromFile()->build();
+
+        // Construct a Google Ads client configured from a properties file and the
+        // OAuth2 credentials above.
+        $googleAdsClient = (new GoogleAdsClientBuilder())
+            ->fromFile()
+            ->withOAuth2Credential($oAuth2Credential)
+            ->build();
+
+        try {
+            self::runExample(
+                $googleAdsClient,
+                $options[ArgumentNames::CUSTOMER_ID] ?: self::CUSTOMER_ID,
+                $options[ArgumentNames::CONVERSION_ACTION_ID] ?: self::CONVERSION_ACTION_ID,
+                $options[ArgumentNames::GCL_ID] ?: self::GCL_ID,
+                $options[ArgumentNames::CONVERSION_TIME] ?: self::CONVERSION_TIME,
+                $options[ArgumentNames::CONVERSION_VALUE] ?: self::CONVERSION_VALUE
+            );
+        } catch (GoogleAdsException $googleAdsException) {
+            printf(
+                "Request with ID '%s' has failed.%sGoogle Ads failure details:%s",
+                $googleAdsException->getRequestId(),
+                PHP_EOL,
+                PHP_EOL
+            );
+            foreach ($googleAdsException->getGoogleAdsFailure()->getErrors() as $error) {
+                /** @var GoogleAdsError $error */
+                printf(
+                    "\t%s: %s%s",
+                    $error->getErrorCode()->getErrorCode(),
+                    $error->getMessage(),
+                    PHP_EOL
+                );
+            }
+        } catch (ApiException $apiException) {
+            printf(
+                "ApiException was thrown with message '%s'.%s",
+                $apiException->getMessage(),
+                PHP_EOL
+            );
+        }
+    }
+
+    /**
+     * Runs the example.
+     *
+     * @param GoogleAdsClient $googleAdsClient the Google Ads API client
+     * @param int $customerId the customer ID
+     * @param int $conversionActionId the ID of the conversion action to upload to
+     * @param string $gclId the GCLID for the conversion (should be newer than 30 days)
+     * @param string $conversionTime the date and time of the conversion (should be after the
```

Comment:
Are there any links to format of datetime that we can use?

Reply:
Added the format information